java/publish/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM maven:3.5-jdk-10 as maven
+FROM maven:3.6.3-jdk-11 as maven
```

Comment:
Is this necessary for this change?

Reply:
the 3.5 (which is an old image) was no longer building due to an old certificate chain and so the `apt-get` were failing